kratos/utilities/geometry_tester.cpp

```diff
@@ -567,6 +584,9 @@ bool GeometryTesterUtility::StreamTestPrism3D6N(
     VerifyStrainExactness( geometry, GeometryData::IntegrationMethod::GI_GAUSS_4, rErrorMessage);
     VerifyStrainExactness( geometry, GeometryData::IntegrationMethod::GI_GAUSS_5, rErrorMessage);
 
+    array_1d<double,3> point_in(3,1.0/3.0);
+    if( !VerifyShapeFunctionsSecondDerivativesValues(geometry,point_in,rErrorMessage) ) successful = false;
```

Comment:
This function is causing me to trigger: 

```Bash
    TestGeometryTester Failed with message: 
        Error: Calling base class ShapeFunctionsSecondDerivatives method instead of derived class one. Please check the definition of derived class. 3 dimensional prism with six nodes in 3D space
    Dimension               : 3
    working space dimension : 3
    Local space dimension   : 3

	Point 1	 :  (0, 0, 0)
	Point 2	 :  (0.333333, 0, 0)
	Point 3	 :  (0, 0.333333, 0)
	Point 4	 :  (0, 0, 0.333333)
	Point 5	 :  (0.333333, 0, 0.333333)
	Point 6	 :  (0, 0.333333, 0.333333)
	Center	 :  (0.111111, 0.111111, 0.166667)

    Jacobian in the origin	 : [3,3]((0.333333,0,0),(0,0.333333,0),(0,0,0.333333))

```

Reply:
That is because the prism_3d_6.h does not have the second derivatives implemented, I do not know why this call is in this function.